modules/rosa-enable-private-cluster-existing.adoc

```diff
@@ -0,0 +1,29 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_aws_private_connections/rosa-private-cluster.adoc
+
+
+[id="rosa-enabling-private-cluster-existing_{context}"]
+= Enabling private cluster on an existing cluster
+
+[role="_abstract"]
+After a cluster has been created, you can later enable the cluster to be private.
+
+.Prerequisites
+
+AWS VPC Peering, VPN, DirectConnect, or TransitGateway has been configured to allow private access.
```

Comment:
We probably want this to be some sort of highlighted section because these are critical prerequisite steps.

Reply:
Well, all prerequisites are critical by nature. The OSD topics that cover how to implement the prerequisites will be included in the same "book" as these ROSA topics, so it all will be in context once it's published. The "book" TOC will be similar to the OSD 4 documentation.